docs/samples/python/example_csv.py

```diff
@@ -0,0 +1,117 @@
+import csv
+import pprint
+import collections
+from typing import List, Iterable, Optional, Union, Dict
+
+from presidio_analyzer import AnalyzerEngine, BatchAnalyzerEngine, RecognizerResult, DictAnalyzerResult
+from presidio_anonymizer import AnonymizerEngine
+from presidio_anonymizer.entities import EngineResult
+
+"""
+Example implementing a CSV analyzer
+
+This example shows how to use the Presidio Analyzer and Anonymizer
+to detect and anonymize PII in a CSV file.
+It uses the BatchAnalyzerEngine to analyze the CSV file, and the
+BatchAnonymizerEngine (defined in batch_processing sample) to anonymize
+the requested columns.

```

Comment:
@navalev maybe we should add a link to the batch notebook? Users might get to this sample first, and the other is less technical but provides more explanation into the process. WDYT?

Reply:
done